File: python/find_unique_binary_string_1980.py
class Solution:
    def findDifferentBinaryString(self, nums: List[str]) -> str:
        n = len(nums)
        res = ['0'] * n
        for i in range(n):
            res[i] = '1' if nums[i][i] == '0' else '0'
        return ''.join(res)

File: rust/src/find_unique_binary_string_1980.rs
use crate::Solution;

impl Solution {
    pub fn find_different_binary_string(nums: Vec<String>) -> String {
        let n = nums.len();
        let mut res = vec!['0'; n];
        for i in 0..n {
            res[i] = if nums[i].as_bytes()[i] == b'0' { '1' } else { '0' };
        }
        res.into_iter().collect()
    }
}
